docker/Dockerfile

```diff
@@ -30,11 +28,10 @@ RUN mkdir /opt/spm12 \
 RUN octave --no-gui --eval "addpath(\"/opt/spm12\"); savepath ();"
 
 # get the CPP BIDS pipeline code
-RUN mkdir /CPP_BIDS_pipeline /output /code \
- && git clone https://github.com/cpp-lln-lab/CPP_BIDS_SPM_pipeline.git /CPP_BIDS_pipeline/
-
-RUN octave --no-gui --eval "addpath(genpath(\"/CPP_BIDS_pipeline\")); savepath ();"
+RUN mkdir /CPP_SPM /output /code \
+ && git clone https://github.com/cpp-lln-lab/CPP_SPM.git /CPP_SPM/
```

Comment:
so, on this line, it clones the whole repo, right? Would not be better to make it fix to a release instead?

Reply:
yes indeed! 

and actually make the clone recursive so we get the submodule

a possibility to get a specific release would be to just `git checkout` the tag of that realease.